### ISSUE TITLE
Grid x without 100% width 

### ISSUE THREAD

**Issue by jgomesmv** [zurb/foundation-sites#11794]
## What should happen?

The grid-x doesn't occupy the entire available space from the parent element.
The 

## What happens instead?

This styles are missing from the xy-grid/_classes.scss file: 
.grid-x {
    margin: 0;
    padding: 0;
    width: 100%;
}

## Test Case and/or Steps to Reproduce (for bugs)
How to reproduce:
1. Install foundation sites
2. Use the class grid-x
3. Check that the element doesn't occupy the entire space of the parent element

## Context
![image](https://user-images.githubusercontent.com/8100887/61950664-7bebe580-afa6-11e9-8729-328b7ca56efd.png)

## Your Environment
- Foundation version(s) used: 6.5.3
- Browser(s) name and version(s): Chrome Version 75.0.3770.142 (Official Build) (64-bit)
- Device, Operating System and version: Windows 10
- Tech Stack: Angular 8.1.2

**Comment by SassNinja** [comment]
Hi @jgomesmv,

I'm unable to reproduce the issue – please provide a codepen

**Comment by DanielRuf** [comment]
I guess it needs the grid frame / container.

**Comment by jgomesmv** [comment]
You have created a small example with foundation on top of a angular app:
https://stackblitz.com/github/jgomesmv/foundation-sites-demo?file=src%2Fstyles%2Fglobal%2F_imports.scss

I have created this stackblitz example because I think the problem it's about how angular loads foundation-sites. It's the second time I'm using foundation-sites, and none of this issues happen before.
Event stackblitz has a problem running the solution it really doesn't care about how I have configured the angular.json file and the npm dependencies. Probably you will need to install the angular cli and run it locally
